packages/@storefront/structure/test/unit/carousel.ts

```diff
@@ -37,12 +37,22 @@ suite('Carousel', ({ expect, spy, stub, itShouldProvideAlias }) => {
     describe('onMount()', () => {
       it('should add event listener for window resize', () => {
         const addEventListener = spy();
+        carousel.set = () => null;
```

Comment:
Since we swapped out `this.set()` for `this.forceUpdate()`, this override should also be updated.